syntax = "proto3";
package Requests;

// Request for a new file
message FileLoadRequest {
    string filename = 1;
}